Release notes for v5.0.8 and v4.6.30 (#3219)
ezsystems/developer-documentation: docs/release_notes/ibexa_dxp_v5.0.md

- **The Personal Shopping Assistant (additional sales)** encourages additional purchases by suggesting complementary cross-selling items
- **The Personal Shopping Assistant (conversion)** helps users discover better product matches by suggesting similar items based on their activity

For more information, see [recommendation blocks](https://doc.ibexa.co/en/5.0/recommendations/raptor_integration/recommendation_blocks/).

#### Cookie lifetime configuration

A new `cookie_id_lifetime_days` configuration option controls the lifetime in days of the server-side tracking identifier cookie.

For more information, see [connector installation and configuration](https://doc.ibexa.co/en/5.0/recommendations/raptor_integration/connector_installation_configuration/).

### Anonymous user segmentation in [[= product_name_cdp =]] [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]

[[= product_name_cdp =]] can now build audiences for anonymous visitors, enabling personalized experiences for users who are not logged in.

When an anonymous visitor accesses your site, Raptor builds an anonymous profile based on tracking data.

> You can segment these profiles into audiences and use them in [[= product_name =]] to deliver personalized content, exactly as with logged-in users.

Reviewer: ```suggestion
[[= product_name_cdp =]] can now build audiences for anonymous visitors.
Use them in [[= product_name =]] to deliver personalized experiences even before users log in.
```

<div class="release-notes" markdown="1">

[[% set version = 'v5.0.8' %]]

[[= release_note_entry_begin(
    'MCP Servers ' + version,
    '2026-05-21',
    ['Headless', 'Experience', 'Commerce', 'LTS Update', 'New feature', 'First release']
) =]]

> MCP servers ease the discovery of interactions with [[= product_name =]] by AI agents.

Reviewer 2: ```suggestion
MCP servers make it easier for AI agents to discover the available interactions with [[= product_name =]].
```

> This feature allows you to configure several MCP servers with their own set of tools.

Reviewer 2: ```suggestion
With the MCP Servers feature, you can configure multiple MCP servers with their specific sets of tools.
```

- **The Personal Shopping Assistant (additional sales)** encourages additional purchases by suggesting complementary cross-selling items
- **The Personal Shopping Assistant (conversion)** helps users discover better product matches by suggesting similar items based on their activity

For more information, see [recommendation blocks](https://doc.ibexa.co/en/5.0/recommendations/raptor_integration/recommendation_blocks/).

#### Cookie lifetime configuration

A new `cookie_id_lifetime_days` configuration option controls the lifetime in days of the server-side tracking identifier cookie.

For more information, see [connector installation and configuration](https://doc.ibexa.co/en/5.0/recommendations/raptor_integration/connector_installation_configuration/).

### Anonymous user segmentation in [[= product_name_cdp =]] [[% include 'snippets/experience_badge.md' %]] [[% include 'snippets/commerce_badge.md' %]]

[[= product_name_cdp =]] can now build audiences for anonymous visitors, enabling personalized experiences for users who are not logged in.

When an anonymous visitor accesses your site, Raptor builds an anonymous profile based on tracking data.

> You can segment these profiles into audiences and use them in [[= product_name =]] to deliver personalized content, exactly as with logged-in users.

Reviewer 2: @mnocon Isn't that too much of an explanation of how the feature works?

Reviewer: You're right, how about:

```
[[= product_name_cdp =]] can now build audiences for anonymous visitors, enabling personalized experiences even before users log in.

Use them in [[= product_name =]] to deliver personalized experiences even before users sign in.
```
Without mentioning "anonymous profiles" etc.

What do you think?

Reviewer 2: The second sentence basically repeats the second part of the first sentence 8-)

How about:
```
[[= product_name_cdp =]] can now build audiences for anonymous visitors.
Use them in [[= product_name =]] to deliver personalized experiences even before users log in.
```

Reviewer: Added in https://github.com/ibexa/documentation-developer/pull/3219/changes/c02dfec4bd2c482f85fa7849c45a00f7f3cec58b (FYI @adriendupuis )

For more information, see [Anonymous user segmentation](https://doc.ibexa.co/en/5.0/cdp/cdp_activation/cdp_configuration/#anonymous-user-segmentation).

### Gaussian blur optimization in Image Editor

> The [Image Editor]([[= user_doc =]]/image_management/edit_images/) now supports configurable gaussian blur strength for image optimization.

Reviewer 2: ```suggestion
The [Image Editor]([[= user_doc =]]/image_management/edit_images/) now supports configuring the strength of the gaussian blur that is used for image optimization.
```
